testpilot/metrics/views.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.conf import settings
+from django.http import Http404
+
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.response import Response
+from rest_framework.decorators import permission_classes, api_view
+
+DEFAULT_LOG_PING_WHITELIST = [
+    'testpilottest'
+]
+
+
+@csrf_exempt
+@api_view(['POST'])
+@permission_classes([])
+def log_ping(request, logger_name):
+    """Accept and log metrics pings"""
+    whitelist = getattr(settings, 'LOG_PING_WHITELIST', DEFAULT_LOG_PING_WHITELIST)
```

Comment:
So we will need to add logger names for each experiment that uses this? I.e., for 404s and my "blok" add-on/experiment, we would need to have:
`settings.py`:
```
LOG_PING_WHITELIST = [
    'blok',
    'nomore404s'
]
```
?